From: Håvard F. Aasen <havard.f.aasen@pfft.no>
Date: Sat, 23 Dec 2023 11:48:16 +0100
Subject: Use version specific 'python3-config'

Needed since we want to build for multiple versions of Python 3.

Forwarded: not-needed
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1ee66bd..4ecf4e9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -182,9 +182,9 @@ else
 	echo "Python3 bindings WILL be built"
 	use_python3=yes
 	AM_PATH_PYTHON([3.1],, [:])
-	PYTHON3_CFLAGS=`python3-config --cflags 2> /dev/null`
-	PYTHON3_LIBS=`python3-config --libs 2> /dev/null`
-	PYTHON3_INCLUDES=`python3-config --includes 2> /dev/null`
+	PYTHON3_CFLAGS=`$PYTHON-config --cflags 2> /dev/null`
+	PYTHON3_LIBS=`$PYTHON-config --libs 2> /dev/null`
+	PYTHON3_INCLUDES=`$PYTHON-config --includes 2> /dev/null`
 	AC_SUBST(PYTHON3_CFLAGS)
 	AC_SUBST(PYTHON3_LIBS)
 	AC_SUBST(PYTHON3_INCLUDES)
